index.js

```diff
@@ -5,8 +5,8 @@ async function fillSheet() {
     const users = await googleSheets.getUsers();
 
     const commits = [];
-    for (let user of users) {
-        commits.push(await gitHub.getCommitsSince(user[0], "one-week-ago"));
+    for (let i = 0; i < users.length; i++) {
+        commits.push(await gitHub.getCommitsSince(users[i][0], i, users.length));
```

Comment:
(I realised the last bit doesn't quite apply with `await`; you can use `Promise.all` but it would try to run things in parallel which isn't what we want here.)